testing/test_driver.py

```diff
@@ -115,3 +115,37 @@ def test_pass(driver_kwargs):
                      '--driver', 'Remote',
                      '--port', port,
                      file_test, passed=1)
+
+
+def test_driver_lowercase(testdir):
+    file_test = testdir.makepyfile("""
+        import pytest
+        @pytest.mark.nondestructive
+        def test_pass(request):
+            assert request.config.getoption('driver') == 'saucelabs'
+    """)
+    testdir.quick_qa('--driver', 'SauceLabs',
+                     file_test, passed=1)
+
+
+def test_provider_lowercase():
```

Comment:
Let's parameterise this test.

Reply:
I thought about it, but I'm not sure of how to accomplish that with the imports... 🤔

Reply:
yeah, I need help with this @davehunt  (the imports).